test/test_fx.py

```diff
@@ -513,5 +514,41 @@ def forward(self, x):
             assert s in stringed
 
 
+    def test_scriptmod_sidechannel(self):
+        class Foo(torch.nn.Module):
+            def forward(self, x):
+                return x + torch.rand(3, 4)
+
+        f = Foo()
+        sym_traced = symbolic_trace(f)
+
+        def serialize(gm : GraphModule) -> io.BytesIO():
```

Comment:
Do we want to introduce these two functions to torch.fx, since it will be used widely if we deal with TS models.

Reply:
I think we didn't want to support it officially and instead just use as a temporary workaround. Otherwise it introduces another way of packaging (with 'code.txt') which is not that nice